### PR TITLE
clean markdown up and add Powershell commands

### PR DESCRIPTION
**Pull request by chris-martinelli**

### CODE REVIEW

File: products/cloudflare-one/src/content/connections/connect-apps/configuration/ports-and-ips.md [hunk 1]
```diff
@@ -23,39 +23,71 @@ Below the output of `dig` commands towards the above hostnames:
...
-...
-
 ;; ANSWER SECTION:
 region2.argotunnel.com.	300	IN	A	198.41.200.193
 region2.argotunnel.com.	300	IN	A	198.41.200.233
 region2.argotunnel.com.	300	IN	A	198.41.200.13
 region2.argotunnel.com.	300	IN	A	198.41.200.53
 region2.argotunnel.com.	300	IN	A	198.41.200.113
-
 ...
```
Comment (nmldiegues) [inline products/cloudflare-one/src/content/connections/connect-apps/configuration/ports-and-ips.md:45]: did you mean to remove these `...` from everywhere? there's more forgotten below

File: products/cloudflare-one/src/content/connections/connect-apps/configuration/ports-and-ips.md [hunk 2]
```diff
@@ -23,39 +23,71 @@ Below the output of `dig` commands towards the above hostnames:
...
 ;; ANSWER SECTION:
 api.cloudflare.com.     41      IN      A       104.19.193.29
 api.cloudflare.com.     41      IN      A       104.19.192.29
-
 ...
 ```
 
+On Windows, you can use PowerShell commands if dig is not available:
+
+To test DNS:
```
Comment (nmldiegues) [inline products/cloudflare-one/src/content/connections/connect-apps/configuration/ports-and-ips.md:59]: there's two sentences ending with `:` in a row above

**Review by nmldiegues — COMMENTED** [review]
Thanks for this contribution!

File: products/cloudflare-one/src/content/connections/connect-apps/configuration/ports-and-ips.md (hunk 1 above)
Comment (chris-martinelli) [reply products/cloudflare-one/src/content/connections/connect-apps/configuration/ports-and-ips.md:45]: Ah I think my eyes just missed this. Updated.

**Review by abracchi-tw — APPROVED** [review]

**Review by nmldiegues — APPROVED** [review]

**Pull request merged**